quantum/keymap_extras/sendstring_belgian.h

```diff
@@ -19,68 +19,59 @@
 #pragma once
 
 #include "keymap_belgian.h"
+#include "quantum.h"
 
 // clang-format off
 
 #ifndef USE_PACKED_KEYCODE_LUT
-const bool ascii_to_shift_lut[128] PROGMEM = {
-    0, 0, 0, 0, 0, 0, 0, 0,
-    0, 0, 0, 0, 0, 0, 0, 0,
-    0, 0, 0, 0, 0, 0, 0, 0,
-    0, 0, 0, 0, 0, 0, 0, 0,
-
-    0, 0, 0, 0, 0, 1, 0, 0,
-    0, 0, 1, 1, 0, 0, 1, 1,
-    1, 1, 1, 1, 1, 1, 1, 1,
-    1, 1, 0, 0, 0, 0, 1, 1,
-    0, 1, 1, 1, 1, 1, 1, 1,
-    1, 1, 1, 1, 1, 1, 1, 1,
-    1, 1, 1, 1, 1, 1, 1, 1,
-    1, 1, 1, 0, 0, 0, 0, 1,
-    0, 0, 0, 0, 0, 0, 0, 0,
-    0, 0, 0, 0, 0, 0, 0, 0,
-    0, 0, 0, 0, 0, 0, 0, 0,
-    0, 0, 0, 0, 0, 0, 0, 0
-};
+const bool ascii_to_shift_lut[128] PROGMEM =
```

Comment:
If we're going to maintain two arrays, it'd probably be easier to `#define` the size in a header and depending on whether it's packed we do 16 or 128. The bool can be swapped to `uint8_t` even for the non-packed version, I'd imagine.

Reply:
So, in a header:
```c
#ifndef USE_PACKED_KEYCODE_LUT 
#define LUT_SIZE 128
#else
#define LUT_SIZE 16
#endif
```

...and:
 `const uint8_t ascii_to_shift_lutp[LUT_SIZE] PROGMEM =`

Reply:
(assuming we don't just collapse the both of them and use the packed unconditionally)